stratum/hal/lib/tdi/tdi_id_mapper.cc

```diff
@@ -0,0 +1,285 @@
+// Copyright 2020-present Open Networking Foundation
+// Copyright 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "stratum/hal/lib/tdi/tdi_id_mapper.h"
+
+#include <vector>
+
+#include "absl/strings/match.h"
+#include "nlohmann/json.hpp"
+#include "stratum/glue/gtl/map_util.h"
+#include "stratum/hal/lib/tdi/tdi_constants.h"
+#include "stratum/hal/lib/tdi/macros.h"
+#include "tdi_rt/tdi_rt_defs.h"
```

Comment:
These are DPDK specific header file. Please add a P4OVS_CHANGES wrapper.
The tofino version is tdi_tofino/tdi_tofino_defs.h.